examples/examplevm/examplevm.c

```diff
@@ -9,6 +9,8 @@ struct examplevm
 {
     struct evmc_instance instance;
     int verbose;
+    evmc_trace_callback trace_callback;
```

Comment:
it would be useful to have another callback here to allow the EVM to log text back to the client.

Reply:
instead of a separate callback, actually it could just be added as an additional parameter to `evmc_trace_callback`.

Reply:
The C++ `std::cerr` works. We could add a logger method to capture VM logs nicely on the Host side. But that is different feature and should be done independently.